src/shared/components/studyTagsTooltip/StudyTagsTooltip.scss

```diff
@@ -0,0 +1,50 @@
+/**
+ * This file is part of cBioPortal.
+ *
+ * cBioPortal is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ **/
+
+.rc-tooltip-inner table {
```

Comment:
these styles will affect all rc tooltips.  not sure that's what we want.

Reply:
If this scss file is imported **anywhere** in the codebase, it will be added to the global file and operate globally.  So you need to contextualize these styles either by wrapping them with, say, studyTagTooltip class, or using css modules.

Reply:
@alisman I changed `.rc-tooltip-inner` by `div.json-to-table`, thus now it is specific for this component. I think this is solved now 😃